parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -506,6 +490,7 @@ finally
     Write-Log "CSE ExecutionDuration: $ExecutionDuration. ExitCode: $global:ExitCode"
     Stop-Transcript
 
+    Logs-To-Event -TaskName "AKS.WindowsCSE.cse_main" -TaskMessage "ExitCode: $global:ExitCode. ErrorMessage: $global:ErrorMessage." 

```

Comment:
Need to log the event before Stop-Transcript. Otherwise, it will not be logged to cse logs.